proper_parens.py

```diff
@@ -0,0 +1,11 @@
+def proper_parens(text):
+    open_paren = text.count('(')
+    closed_parens = text.count(')')
+    if open_paren == closed_parens:
+        return 0
+
+    if closed_parens > open_paren:
+        return -1
+
+    if closed_parens < open_paren:
+        return 1
```

Comment:
This implementation pays no attention to the order in which the parens appear in the string.  For example, you rate in  your tests the string ")(" as open, but it is not.  It is broken because a close parens appears before any open parens it can match with.  

Please try again.